### PR TITLE
[plpgsql] Merge pgunit and pgtap

### PR DESCRIPTION
**Pull request by fpellet**

### CODE REVIEW

**Comment by fpellet** [comment]
@GradedJestRisk is ok for you ?  
 Maybe we should add an example or a procedure to make a golden master with pgtap

**Comment by emilybache** [comment]
Thanks! I'm happy to merge this when you have both agreed on it.

**Comment by GradedJestRisk** [comment]
Hi Florian, as stated [here](https://github.com/emilybache/GildedRose-Refactoring-Kata/pull/137#issuecomment-574548864), you PR has erased some code (root folder + pgtap), including golden master, I'm currently creating another PR to fix this.

**Comment by fpellet** [comment]
Hi @GradedJestRisk  
Sorry, I don't understand.
- The PR #137 removes no file (https://github.com/emilybache/GildedRose-Refactoring-Kata/pull/137/files). But this PR, I have restore old files and pgunit works now.
- Tests and source files are separated. There are 3 directories: src (structure rename to src) , pgtab and pgunit. I don't understand your suggestion.
- You can use without docker. The readme indicates the procedure.

Please, can you rephrase your remarks and suggestions ?

**Comment by GradedJestRisk** [comment]
Hi @fpellet. 
Sorry for the month-delay, a private issue prevented me to answer ealier

I'm unsure about what happened. My git skills are not that good, so I can't figure what did happen, or if everything will be allright afterwards. I remember @emilybache warned me about possible merge issues.

I suggest to follow Kent Beck advice: test, rather than talk => we'll see if everything ends well.

Regards,
Pierre

**Comment by fpellet** [comment]
@emilybache it's ok for you? Is there anything missing?

**Comment by emilybache** [comment]
Hi, Sorry, now it's my turn to be slow. I'm happy to trust you that this looks ok now.
